src/minimal-webapi-openidconnect-v2/Program.cs

```diff
@@ -0,0 +1,52 @@
+using Microsoft.AspNetCore.Authentication;
+using Microsoft.AspNetCore.Authentication.JwtBearer;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.Identity.Web;
+
+var builder = WebApplication.CreateBuilder(args);
+
+builder.Services.AddAuthentication(JwtBearerDefaults.AuthenticationScheme)
+                .AddMicrosoftIdentityWebApi(builder.Configuration.GetSection("AzureAd"));
+builder.Services.AddAuthorization();
+
+builder.Services.AddEndpointsApiExplorer();

```

Comment:
can we remove this ones as well?

Reply:
done | addressed from [dde42](https://github.com/Azure-Samples/ms-identity-docs-code-csharp/pull/2/commits/dde42d20cfc0f3c9bd8894468e370cfb6ebcf293)

Reply:
<3